.github/workflows/benchmark.yml

```diff
@@ -12,12 +12,12 @@ jobs:
           [
             "sift_front_end,  door-12,               12,  JPG,  test_data,  olsson-loader,  1296, true",
             "deep_front_end,  door-12,               12,  JPG,  test_data,  olsson-loader,  1296, true",
-            "sift_front_end,  skydio-8,              8,   jpg,  gdrive ,    colmap-loader,  760, true",
-            "deep_front_end,  skydio-8,              8,   jpg,  gdrive,     colmap-loader,  760, true",
-            "sift_front_end,  skydio-32,             32,  jpg,  gdrive,     colmap-loader,  760, true",
-            "deep_front_end,  skydio-32,             32,  jpg,  gdrive,     colmap-loader,  760, true",
-            "sift_front_end,  palace-fine-arts-281,  25,  jpg,  wget,       olsson-loader,  320, true",
-            "deep_front_end,  notre-dame-20,         20,  jpg,  gdrive,     colmap-loader,  760, false",
+            "sift_front_end,  skydio-8,              8,   jpg,  gdrive ,    colmap-loader,  760,  true",
+            "deep_front_end,  skydio-8,              8,   jpg,  gdrive,     colmap-loader,  760,  true",
+            "sift_front_end,  skydio-32,             32,  jpg,  gdrive,     colmap-loader,  760,  true",
+            "deep_front_end,  skydio-32,             32,  jpg,  gdrive,     colmap-loader,  760,  true",
+            "sift_front_end,  palace-fine-arts-281,  25,  jpg,  wget,       olsson-loader,  320,  true",
+            "deep_front_end,  notre-dame-20,         20,  jpg,  gdrive,     colmap-loader,  760,  false",
```

Comment:
extra space needed on the last line as well, and first two lines, since comma followed by two spaces is the delimiter?

Reply:
Oh ok. I thought it was just for prettifying.

Reply:
could you add the extra space for sed to split it?

Reply:
@ayushbaid gentle reminder about this comment